Cubical/Data/Int/MoreInts/QuoInt/Properties.agda

```diff
@@ -6,13 +6,15 @@ open import Cubical.Foundations.Prelude
 open import Cubical.Foundations.Function
 open import Cubical.Foundations.HLevels
 open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.Isomorphism
 open import Cubical.Relation.Nullary
 
 open import Cubical.Data.Nat as ℕ using (ℕ; zero; suc)
 open import Cubical.Data.Bool as Bool using (Bool; not; notnot)
 open import Cubical.Data.Empty
 open import Cubical.Data.Unit renaming (Unit to ⊤)
 
+open import Cubical.Data.Int renaming (_+_ to _+Int_; _·_ to _·Int_; ℤ to Int; sucℤ to sucInt; predℤ to predInt; sucℤ+ to sucInt+; predℤ+ to predInt+) hiding (neg; pos; abs; isSetℤ; -_; _+'_; +'≡+)
```

Comment:
You probably forgot to remove the `open` when importing the module qualified, hence the conflicts. I'd try a `import Cubical.Data.Int as Int`.

Reply:
Ha, I was sure I tried that combination, but perhaps I wanted to have some things imported renamed/without the prefix. I still don't know how to do that, but perhaps it's better not to anyway. <_<"